src/Console/Command/CompareCommand.php

```diff
@@ -198,24 +204,16 @@ private function buildTable(OutputInterface $output)
     {
         $table = new Table($output);
 
-        $columns = ['Fixer', 'Is Configured', 'Is Enabled', 'Is Risky', 'Is Inherited', 'In RuleSet'];
-
-        // Displays the totals about found Fixers
-        $totalsLine = sprintf('Found <fg=yellow;>%s built-in</> fixers. Of those, <fg=yellow;>%s are configured</>, <fg=yellow;>%s are enabled</> and <fg=yellow;>%s are risky</>.', count($this->builtInFixers), count($this->configuredFixers), count($this->enabledFixers), count($this->riskyFixers));
-
-        // Displays the visibility set to filter the shown Fixers
-        $visibilityLine = sprintf(
-            'Hide Configured: %s | Hide Enabled: %s | Hide Risky: %s',
-            $this->hideConfigured ? self::YES : self::NO,
-            $this->hideEnabled ? self::YES : self::NO,
-            $this->hideRisky ? self::YES : self::NO
-        );
+        $columns = [
+            sprintf('Fixer (%s)', count($this->builtInFixers)),
+            sprintf("Configured (%s)\nAre hided: %s", count($this->configuredFixers), $this->hideConfigured ? self::YES : self::NO),
+            sprintf("Enabled (%s)\nAre hided: %s", count($this->enabledFixers), $this->hideEnabled ? self::YES : self::NO),
+            sprintf("Risky (%s)\nAre hided: %s", count($this->riskyFixers), $this->hideRisky ? self::YES : self::NO),
+            sprintf("Inherited (%s)\nAre hided: %s", $this->countInherited, $this->hideInherited ? self::YES : self::NO),
```

Comment:
`hided` isn't really a word, should be `Are hidden`

Reply:
You are right... Fixed!